wayland-wm/sway/sway-9999.ebuild

```diff
@@ -15,35 +15,30 @@ EGIT_REPO_URI="https://github.com/SirCmpwn/sway.git
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS=""
-IUSE=""
+IUSE='systemd'
 
-DEPEND="${RDEPEND}
+DEPEND="
+	${RDEPEND}
+	app-text/asciidoc
 	"
 
 RDEPEND="
-	app-text/asciidoc
 	dev-libs/json-c
 	dev-libs/libpcre
 	dev-libs/wlc
-	x11-base/xorg-server[wayland]"
-
+	x11-base/xorg-server[wayland]
+	x11-libs/cairo
+	x11-libs/libxkbcommon
+	x11-libs/pango
+	systemd? ( sys-apps/systemd )
+	"
 
 src_prepare() {
-	cmake-utils_src_prepare
-}
-
-src_configure() {
-	cmake-utils_src_configure
-}
-
-src_compile() {
-	cmake-utils_src_compile
-}
-
-src_test() {
-	cmake-utils_src_test
+	sed -i 's|set(CMAKE_C_FLAGS "-g")||' CMakeLists.txt || \
+		die 'Unable to remove debug flag'
 }
 
 src_install() {
 	cmake-utils_src_install
+	use !systemd && fperms u+s '/usr/bin/sway'
```

Comment:
It does actually, because I do not have logind to handle privileges for me.

From the sway readme
> On systems without logind, you need to suid the sway binary:
> `sudo chmod a+s /usr/bin/sway`

Actually, I don't know why it grants privileges to all instead of user (`a+s` instead of `u+s`), I was able to launch it without issue with only user.